tests/test_SklearnPolynomialFeaturesConverter.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tests scikit-learn's polynomial features converter.
+"""
+
+import unittest
+import numpy as np
+from sklearn.preprocessing import PolynomialFeatures 
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType, Int64TensorType
+from test_utils import dump_data_and_model
+
+
+class TestSklearnPolynomialFeatures(unittest.TestCase):
+
+    def test_model_polynomial_features_float(self):
+        X = np.array([[1.2, 3.2], [4.3, 3.2], [3.2, 4.7]])
```

Comment:
Did try with vector of 3 coordinates?

Reply:
Yes, added unit tests with more than two features.